languages/clojure/exercises/concept/booleans/.cpcache/2272782656.cp

```diff
@@ -0,0 +1 @@
+src:/home/porky/.m2/repository/com/cognitect/transit-java/0.8.337/transit-java-0.8.337.jar:/home/porky/.m2/repository/org/clojure/clojure/1.10.1/clojure-1.10.1.jar:/home/porky/.m2/repository/commons-codec/commons-codec/1.10/commons-codec-1.10.jar:/home/porky/.m2/repository/io/lambdaforge/datalog-parser/0.1.1/datalog-parser-0.1.1.jar:/home/porky/.m2/repository/org/clojure/core.specs.alpha/0.2.44/core.specs.alpha-0.2.44.jar:/home/porky/.m2/repository/org/clojure/spec.alpha/0.2.176/spec.alpha-0.2.176.jar:/home/porky/.m2/repository/com/fasterxml/jackson/dataformat/jackson-dataformat-cbor/2.9.6/jackson-dataformat-cbor-2.9.6.jar:/home/porky/.m2/repository/com/googlecode/json-simple/json-simple/1.1.1/json-simple-1.1.1.jar:/home/porky/.m2/repository/com/fasterxml/jackson/core/jackson-core/2.9.6/jackson-core-2.9.6.jar:/home/porky/.m2/repository/clj-kondo/clj-kondo/2020.04.05/clj-kondo-2020.04.05.jar:/home/porky/.m2/repository/org/javassist/javassist/3.18.1-GA/javassist-3.18.1-GA.jar:/home/porky/.m2/repository/org/msgpack/msgpack/0.6.12/msgpack-0.6.12.jar:/home/porky/.m2/repository/com/cognitect/transit-clj/0.8.313/transit-clj-0.8.313.jar:/home/porky/.m2/repository/cider/cider-nrepl/0.23.0/cider-nrepl-0.23.0.jar:/home/porky/.m2/repository/cheshire/cheshire/5.8.1/cheshire-5.8.1.jar:/home/porky/.m2/repository/tigris/tigris/0.1.1/tigris-0.1.1.jar:/home/porky/.m2/repository/nrepl/nrepl/0.6.0/nrepl-0.6.0.jar:/home/porky/.m2/repository/com/fasterxml/jackson/dataformat/jackson-dataformat-smile/2.9.6/jackson-dataformat-smile-2.9.6.jar:/home/porky/.m2/repository/javax/xml/bind/jaxb-api/2.3.0/jaxb-api-2.3.0.jar
```

Comment:
I think the files in the `.cpcache` should probably not have been included in this PR?